hq/app/auth/SecurityHQAuthActions.scala

```diff
@@ -21,7 +21,7 @@ trait SecurityHQAuthActions extends LoginSupport {
   override val authConfig: GoogleAuthConfig
 
   val googleGroupChecker: GoogleGroupChecker = Config.googleGroupChecker
-  val requiredGoogleGroups = Set(Config.twoFAGroup)
+  val requiredGoogleGroups: Set[String] = Set(Config.twoFAGroup, Config.departmentGroup)
```

Comment:
The `twoFAGroup` is not properly enforcing 2FA, we can rely on the department group only here I think.

Reply:
Keeping things as they are for now, as per other PRs, but I agree that this makes sense.